test/spec/algoliasearch.helper/queryID.js

```diff
@@ -21,7 +21,7 @@ test('the queryid should keep increasing when new requests arrives', function(t)
   t.end();
 });
 
-test('the response handler should check that the query is not outdated', function(t) {
+test.skip('the response handler should check that the query is not outdated', function(t) {
```

Comment:
Do we still have a test that checks it at a higher level maybe (Not getting too many "results" events given the outdated response).